snippets/facets.liquid

```diff
@@ -372,18 +372,18 @@
             </div>
 
             <noscript>
-              <button type="submit" class="facets__button-no-js button">{{ 'products.facets.sort_button' | t }}</button>
+              <button type="submit" class="facets__button-no-js button button--secondary">{{ 'products.facets.sort_button' | t }}</button>
             </noscript>
           </div>
         </form>
-      <facet-filters-form>
+      </facet-filters-form>
     {%- endif -%}    
   {%- endif -%}
   {% comment %}  Drawer and mobile filter {% endcomment %}
   <menu-drawer class="mobile-facets__wrapper{% if filter_type == 'horizontal' or filter_type == 'vertical' %} medium-hide large-up-hide{% endif %}" data-breakpoint="mobile">
     <details class="mobile-facets__disclosure disclosure-has-popup">
       <summary class="mobile-facets__open-wrapper focus-offset">
-        <span class="mobile-facets__open">
+        <span class="mobile-facets__open{% if filter_type == 'drawer' and enable_filtering == false %} medium-hide large-up-hide{% endif %}">
```

Comment:
This one I cannot change. The menu drawer is used on mobile for all filter types. So in this case, it's checking for a desktop specific layout, which is independent from the mobile drawer.

Reply:
Since it's part of the custom `<menu-drawer` element, I thought it would only apply to the drawer. So if this is coming up on desktop, then it means that the filter type is `drawer`.  

But this being a small nitpick, we can disregard 👍